src/main/java/org/jetbrains/kotlin/utils/utils.kt

```diff
@@ -16,8 +16,5 @@
 
 package org.jetbrains.kotlin.utils
 
-fun <T : Any> T?.singletonOrEmptyList(): List<T> =
-        if (this != null) listOf(this) else emptyList()
-
-inline fun <T, C: Collection<T>> C.ifEmpty(body: () -> C): C =
+inline fun <T, C : Collection<T>> C.ifEmpty(body: () -> C): C =
```

Comment:
Is it ok to have such functions as public here?

Reply:
This code is copied automatically from Kotlin. I guess we can make Kotlin declarations internal by hand, but we can't do this with Java sources.